lambda/main.go

```diff
@@ -146,7 +149,7 @@ func Handler(ctx context.Context, evt json.RawMessage) (string, error) {
 		if err != nil {
 			return "", err
 		}
-		metricsBackend = backend.NewCloudWatchBackend(awsRegion, dimensions)
+		metricsBackend = backend.NewCloudWatchBackend(awsRegion, dimensions, int64(time.Since(lastPollTime).Seconds()), enableHighResolution)
```

Comment:
The first time this runs lastPollTime will be 0 so this will be epoch seconds (i.e. 1720764897). But since we only care if it's less than or more than 60 seconds then that's inconsequential. The first metric we send within this lambda context won't be high frequency, but that should be OK